packages/package-json-lint/src/lint.js

```diff
@@ -2,10 +2,20 @@ const fs = require('fs-extra');
 const rules = require('./rules');
 const PackageIssues = require('./issues/PackageIssues');
 const AggregateIssues = require('./issues/AggregateIssues');
-const { getRuleConfig, getConfig } = require('./config');
+const { getRuleConfig, getConfigForFile } = require('./config');
 const { getPathsForPackages } = require('./project-structure');
 
+/**
+ * Lints a package json file with the given package json data with the config provided by the config parameter
+ * @param {Object} options - options for linting the package json
+ * @param {string} options.packageJsonData - the package json data
+ * @param {Object} options.config - the config to use when running the linter
```

Comment:
Looks like the documentation here and line 44 are not accurate. The functions should return an _array_ of issues.